.gitignore

```
@@ -3,4 +3,5 @@
 .vscode/
 
 # Local testing for bluebuild recipe files
-/config/
+/config/*
+Containerfile
```

GitHub complaining about not ending the file with a newline.
```suggestion
Containerfile

```